website/registrations/tests/test_admin.py

```diff
@@ -471,16 +466,16 @@ def test_download_csv__post_different_semesters(self):
 
         old_semester = Semester.objects.create(year=2000, season=Semester.SPRING)
 
-        Registration.objects.create(
+        reg = Registration.objects.create(
             user=user_different_semester,
-            project=self.project,
             semester=old_semester,
             dev_experience=Registration.EXPERIENCE_BEGINNER,
             preference1=self.project,
             course=self.course,
             comments="comment",
             is_international=False,
         )
+        reg.project = self.project
 
```

Comment:
Should add test for 2 projects, can be a separate issue

Reply:
Meh I agree, it's probably wise to make a test case in this PR (to keep things atomic and the git history clean)